aws/resource_aws_ssm_document_test.go

```diff
@@ -383,13 +385,15 @@ func testAccAWSSSMDocumentBasicConfig(rName string) string {
 resource "aws_ssm_document" "foo" {
   name = "test_document-%s"
 	document_type = "Command"
+  tags = {
+           Name = "My Document"
+         }
```

Comment:
Nitpick: Do you mind fixing the indentation here? Should be just 2 spaces (which applies to the above field `document_type`).
Unfortunately we don't have any automated checks for this (yet), but I'm hoping in the future we could `terraform fmt` all the configs in acceptance tests of all providers 😄